TSE/CCExtensions.cpp

```diff
@@ -1656,18 +1656,34 @@ static PRIMITIVEPROCDEF g_Extensions[] =
 			"property (all)\n\n"
 
 			"   'ascended\n"
-			"   'category\n"
+			"   'category -> 'beam | 'effect | 'marker | 'missile | 'ship | 'station\n"
+			"   'commsKey\n"
+			"   'currency -> currency type UNID\n"
```

Comment:
I would love to come up with something more consistent, but I'm happy to take this PR since it is better than what we've got now.

In shpSetAISettings I use three different formats(!) for describing parameters:

FORMAT 1:
'combatStyle
   = 'advanced
   = 'chase
   ...

FORMAT 2:
'aggressor (True/Nil)

FORMAT 3:
'combatSeparation {pixels}

Let's come up with a more consistent format (maybe in the forums).